spiffworkflow-frontend/src/components/ProcessInstanceListTable.tsx

```diff
@@ -1,1653 +1,228 @@
-      });
-      setReportMetadata(reportMetadataCopy);
-      setReportColumnToOperateOn(null);
-      setShowReportColumnForm(false);
-      setRequiresRefilter(true);
-    }
-  };
-
-  const reportColumnToReportColumnForEditing = (reportColumn: ReportColumn) => {
-    const reportColumnForEditing: ReportColumnForEditing = Object.assign(
-      reportColumn,
-      { filter_field_value: '', filter_operator: '' }
-    );
-    if (reportColumn.filterable) {
-      const reportFilter = getFilterByFromReportMetadata(
-        reportColumnForEditing.accessor
-      );
-      if (reportFilter) {
-        reportColumnForEditing.filter_field_value =
-          reportFilter.field_value || '';
-        reportColumnForEditing.filter_operator =
-          reportFilter.operator || 'equals';
-      }
-    }
-    return reportColumnForEditing;
-  };
-
-  const updateReportColumn = (event: any) => {
-    let reportColumnForEditing = null;
-    if (event.selectedItem) {
-      reportColumnForEditing = reportColumnToReportColumnForEditing(
-        event.selectedItem
-      );
-    }
-    setReportColumnToOperateOn(reportColumnForEditing);
-    setRequiresRefilter(true);
-  };
-
-  // options includes item and inputValue
-  const shouldFilterReportColumn = (options: any) => {
-    const reportColumn: ReportColumn = options.item;
-    const { inputValue } = options;
-    return (
-      !reportColumnAccessors().includes(reportColumn.accessor) &&
-      (reportColumn.accessor || '')
-        .toLowerCase()
-        .includes((inputValue || '').toLowerCase())
-    );
-  };
-
-  const setReportColumnConditionValue = (event: any) => {
-    if (reportColumnToOperateOn) {
-      const reportColumnToOperateOnCopy = {
-        ...reportColumnToOperateOn,
-      };
-      reportColumnToOperateOnCopy.filter_field_value = event.target.value;
-      setReportColumnToOperateOn(reportColumnToOperateOnCopy);
-      setRequiresRefilter(true);
-    }
-  };
+  // eslint-disable-next-line sonarjs/no-duplicate-string
+  let processInstanceApiSearchPath = '/process-instances/for-me';
+  if (variant === 'all') {
+    processInstanceApiSearchPath = '/process-instances';
+  }
 
-  const setReportColumnConditionOperator = (selectedItem: string) => {
-    if (reportColumnToOperateOn) {
-      const reportColumnToOperateOnCopy = {
-        ...reportColumnToOperateOn,
-      };
-      const filterOperator = filterOperatorMappings[selectedItem];
-      reportColumnToOperateOnCopy.filter_operator = filterOperator.id;
-      setReportColumnToOperateOn(reportColumnToOperateOnCopy);
-      setRequiresRefilter(true);
+  // Useful to stop refreshing if an api call gets an error
+  // since those errors can make the page unusable in any way
+  const clearRefreshRef = useRef<any>(null);
+  const stopRefreshing = useCallback((error: any) => {
+    if (clearRefreshRef.current) {
+      clearRefreshRef.current();
     }
-  };
+    if (error) {
+      console.error(error);
+    }
+  }, []);
```

Comment:
We mostly want the auto refreshing to stop somewhat silently since the impact is somewhat negligible but some ability to debug is nice hence the console.error.